games-arcade/rocksndiamonds/rocksndiamonds-4.0.0.2.recipe

```diff
@@ -48,14 +48,15 @@ BUILD_PREREQUIRES="
 
 BUILD()
 {
-	make sdl RW_GAME_DIR=$settingsDir/rocksndiamonds
+	make #RW_GAME_DIR=$settingsDir/rocksndiamonds
```

Comment:
We should move the settings to home/config/settings/rocksndiamonds rather than home/.rocksndiamonds still, right?